minecraft_servers/teammlg/manifest.json

```diff
@@ -0,0 +1,70 @@
+{
+	"server_name": "teammlg",
+	"nice_name": "TeamMLG",
+	"direct_ip": "teammlg.net",
+	"server_wildcards": [
+		"%.teammlg.net"
+	],
+	"supported_languages": [
+		"de",
+		"en"
+	],
+	"social": {
+		"web": "https://teammlg.net",
+		"web_shop": "https://store.teammlg.net",
+		"web_support": "https://teammlg.net/forum",
```

Comment:
Website not available.

Reply:
Could you please push it? The website will go online in 1-2 weeks!

Reply:
No, please remove. You can create a new PR THEN. Usually, we push PRs fast, so there is no reason to already show a dead link.